data/mods/DinoMod/monsters/dinosaur.json

```
@@ -1251,5 +1251,54 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT" ],
     "placate_triggers": [ "MEAT" ],
     "categories": [ "DINOSAUR" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_qianzhousaurus",
+    "name": { "str_sp": "Qianzhousaurus" },
+    "species": "DINOSAUR",
+    "default_faction": "qianzhousaurus",
+    "symbol": "d",
+    "color": "white_brown",
+    "volume": "5000 L",
+    "weight": "767 kg",
+    "bodytype": "elephant",
+    "material": [ "flesh" ],
+    "aggression": 300,
+    "morale": 80,
+    "speed": 110,
+    "dodge": 6,
+    "melee_skill": 10,
+    "melee_dice": 10,
+    "melee_dice_sides": 8,
+    "melee_cut": 16,
```

```suggestion
    "melee_cut": 12,
```
More in line with other dinos